packages/common/src/api/account.ts

```diff
@@ -141,8 +141,7 @@ const accountApi = createApi({
               state.userManagers.push({
                 grant: {
                   created_at: currentTime,
-                  // TODO(nkang - C-4332) - Fill this in
-                  grantee_address: '',
+                  grantee_address: managerUser.erc_wallet,
```

Comment:
I continue to wonder which one of these we are supposed to use? I've seen `erc_wallet` not be defined all the time, while `wallet` is?